packages/components/date-picker/src/date-picker-com/panel-date-range.ts

```diff
@@ -0,0 +1,165 @@
+import { computed, inject, nextTick, ref } from 'vue'
+import { useLocale } from '@element-plus/hooks'
+import {
+  getValidDateOfMonth,
+  getValidDateOfYear,
+} from '@element-plus/components/date-picker/src/utils'
```

Comment:
```suggestion
} from '../utils'
```
Relative paths can be used within the same component.